includes/controls/customizer-upsell-control.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+if ( class_exists( 'WP_Customize_Control' ) ) {
```

Comment:
Move logic to the include area

Reply:
This `if` statement should not be inside this file. It should be used to condition the loading of this file. This file should not at all be `included` if the `WP_Customize_Control` class doesn't exist. It makes a lot of difference in performance.

Reply:
Moved the logic to the include area, also updated how it loads and removed redundant checks (we had some duplication on the checks)